src/mail-app/mail/model/MailModel.ts

```diff
@@ -412,11 +412,11 @@ export class MailModel {
 	}
 
 	canManageLabels(): boolean {
-		return this.logins.getUserController().isInternalUser() && this.logins.isEnabled(FeatureType.Labels)
+		return this.logins.getUserController().isInternalUser() //&& this.logins.isEnabled(FeatureType.Labels)
```

Comment:
Was this temporarily commented out?

Reply:
Yes it is temporarily commented to test labels. I will revert it.